client/src/components/overview/DefaultView.jsx

```diff
@@ -1,29 +1,76 @@
+import { ProductDetailsContainer } from '../styles/Overview/ProductDetails.styled.js';
 import { DefaultViewContainer, BackgroundImageContainer, BackgroundImage, ThumbnailSliderContainer } from '../styles/Overview/DefaultView.styled';
 import { Arrow, FadedArrow, ActiveThumbnail, FadedThumbnail, ThumbnailContainer, SliderContainer } from '../styles/Overview/Slider.styled';
 
-export const DefaultView = () => {
+export const DefaultView = (props) => {
+  const [ styles, setStyles ] = useState([]);
+  const [ selectedStyle, setSelectedStyle ] = useState(props.styles[0].style_id);
+  const [ expandedView, setExpandedView ] = useState(false);
   const { images, setImages, currentImage, handleKeyPress } = useContext(OverviewContext);
   const context = useContext(OverviewContext);
 
-  useEffect(() => {
-    getProductPhotosOfAStyle(39333, 234004)
-      .then(results =>
-      setImages(results.photos.map((photo, id) => {return {id, ...photo}}))
-    );
+  useEffect( () => {
+    getProductPhotosOfAStyle(props.productId, selectedStyle)
+      .then( (results) => {
+        setImages(results.photos.map((photo, id) => {
+          return {id, ...photo}}
+        ));
+      })
+      .catch( (err) => {
+        console.error(err);
+      });
   }, []);
 
+  const handleImageClick = () => {
+    setExpandedView(true);
+  }
+
+  const updateStyle = (styleId) => {
+    setSelectedStyle(styleId);
+    getProductPhotosOfAStyle(props.productId, styleId)
+      .then( (results) => {
+        setImages(results.photos.map((photo, id) => { return {id, ...photo}} ));
+      })
+      .catch( (err) => {
+        console.error(err);
+      });
+  }
 
   return (
-    <DefaultViewContainer className='default-view-container' tabIndex="0" onKeyUp={(e) => { e.preventDefault(); handleKeyPress(e);}}>
+    <DefaultViewContainer className='default-view-container'
+      tabIndex='0'
+      onKeyUp={(e) => { e.preventDefault(); handleKeyPress(e); }} >
       <ThumbnailSliderContainer className='thumbnail-slider-container'>
         <ThumbnailSlider />
       </ThumbnailSliderContainer>
       <BackgroundImageContainer className='background-img-container'>
-        {images.length && <BackgroundImage src={images[currentImage]['url']} />}
+        {images.length && <BackgroundImage src={images[currentImage].url} onClick={handleImageClick} />}
       </BackgroundImageContainer>
+      <ProductDetailsContainer>
+        <ProductDetails
+          product={props.product}
+          productId={props.productId}
+          rating={props.rating}
+        />
+        <br></br>
+        <Selectors
+          productId={props.productId}
+          styles={props.styles}
+          addItemToCart={props.addItemToCart}
+          updateStyle={updateStyle}
+        />
+      </ProductDetailsContainer >
+      { expandedView ? <ModalForm submitInModal={false} toggleModal={() => {setExpandedView(false)}} >
+        <ExpandedView src={images[currentImage].url} />
+      </ModalForm> : null}
```

Comment:
Oh, I made the modal to be a form, that may not be what you want to use for `expandedView`. We can change Modal to be even more general of a component without headers, footers or forms but we likely need to discuss that as a team since we all are using it.